### PR TITLE
PMM-4504 Build dist on packaging.

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-497-d8aa378
client docker - perconalab/pmm-client-fb:PR-497-d8aa378
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-497-d8aa378.tar.gz

File: build/bin/build-server-rpm [hunk 1]
```diff
@@ -238,6 +238,19 @@ d }\" \
 
                 spectool -C SOURCES -g SOURCES/${spec_name}.spec
 
+                rpmbuild --define '_rpmdir %{_topdir}/RPMS/${spec_name}-${rpm_version}' --define 'dist .el7' -ba SOURCES/${spec_name}.spec
+                rm -f SOURCES/${spec_name}.spec*
+	    elif [ \"${spec_name}\" == \"percona-qan-app\" ]; then
+	        sudo yum remove -y nodejs npm
```
Comment (AlekSi) [inline build/bin/build-server-rpm]: There seems to be some whitespace problem.

Comment (BupycHuk) [reply build/bin/build-server-rpm]: fixed

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-497-8a66660
client docker - perconalab/pmm-client-fb:PR-497-8a66660
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-497-8a66660.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-497-95b0efa
client docker - perconalab/pmm-client-fb:PR-497-95b0efa
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-497-95b0efa.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-497-9334026
client docker - perconalab/pmm-client-fb:PR-497-9334026
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-497-9334026.tar.gz

File: build/bin/build-server-rpm [hunk 2]
```diff
@@ -227,19 +227,19 @@ d }\" \
...
-                sudo /usr/bin/curl --silent --location https://rpm.nodesource.com/setup_10.x | sudo bash -
-                sudo yum clean all
-                sudo yum -y install nodejs
-                sudo sed -i 's|SRPMS|x86_64|g' /etc/yum.repos.d/nodesource-el7.repo
+	    elif [ \"${spec_name}\" == \"grafana\" ] || [ \"${spec_name}\" == \"percona-qan-app\" ]; then
+	            sudo yum remove -y nodejs npm
+              sudo /usr/bin/curl --silent --location https://rpm.nodesource.com/setup_10.x | sudo bash -
+              sudo yum clean all
+              sudo yum -y install nodejs
+              sudo sed -i 's|SRPMS|x86_64|g' /etc/yum.repos.d/nodesource-el7.repo
```
[the remark this answers is not in the document]
Comment (BupycHuk) [reply build/bin/build-server-rpm]: i don't know, I've just fixed indents

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-497-c276baf
client docker - perconalab/pmm-client-fb:PR-497-c276baf
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-497-c276baf.tar.gz

**Pull request merged**